relay-server/src/services/outcome_aggregator.rs

```diff
@@ -155,7 +151,6 @@ impl OutcomeAggregator {
                 quantity,
             };
 
-            relay_log::trace!("Flushing outcome for timestamp {timestamp}");
```

Comment:
I removed these because they produce a lot of noise when integration testing locally.